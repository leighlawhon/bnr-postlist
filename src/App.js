import React from 'react';
import PostList from './features/postlist/PostList.tsx';

import './App.css';

export default function App() {
  return (
    <div className="App">
      <header>

      </header>
      <main className="">
        <PostList />
      </main>
    </div>
  );
}


