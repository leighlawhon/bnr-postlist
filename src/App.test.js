import React from "react";
import App from "./App";
import { render as rtlRender } from '@testing-library/react'


jest.mock('./features/postlist/PostList', () => () => (<div>Hello World</div>));

test('renders', () => {
  const { container } = rtlRender(<App />);
  expect(container.textContent)
    .toMatch('Hello World');
});

