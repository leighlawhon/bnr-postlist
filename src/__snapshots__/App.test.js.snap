// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My App Component should render 1`] = `
<div
  className="App"
>
  <header />
  <main
    className=""
  >
    <div
      className="container"
    >
      <div
        className="featured"
      >
        <div>
          <h2
            data-testid="featured-author"
          >
            Featured Author: 
            <select
              onChange={[Function]}
            />
          </h2>
          <ul />
        </div>
      </div>
      <div
        className="other"
      >
        <div>
          <h2>
            Other Posts
          </h2>
          <ul />
        </div>
      </div>
    </div>
  </main>
</div>
`;
